ha_3scale.md

```diff
@@ -0,0 +1,63 @@
+# Configure HA
+
+This section explains how to install 3scale using the
+[3scale-operator](https://github.com/3scale/3scale-operator) and how to
+configure it to have a highly-available deployment.
+
+The first thing that you need to do is install the operator. To do so, you can
+follow [this
+guide](https://github.com/3scale/3scale-operator/blob/3scale-2.9-stable/doc/operator-user-guide.md#installing-3scale).
+
+After installing the operator, we need to deploy an `APIManager` custom
+resource. The operator will read that resource and deploy 3scale configured
+according to its spec. There's a basic example of an `APIManager`
+[here](https://github.com/3scale/3scale-operator/blob/3scale-2.9-stable/doc/operator-user-guide.md#basic-installation).
+
+In order to set up an HA deployment we need to tune a few things in the
+`APIManager` spec:
+- We need to tell the Operator not to deploy the critical DBs. The Operator can
+deploy those DBs, but it cannot set them up to offer HA. To do that, follow
+[these
+instructions](https://github.com/3scale/3scale-operator/blob/3scale-2.9-stable/doc/operator-user-guide.md#external-databases-installation).
+In order to configure HA for those DBs, please refer to [this other
```

Comment:
As a minor comment, since 3scale 2.10 zync DB can also be deployed externally as there are some edge cases where it was identified might be good to have it available. See https://issues.redhat.com/browse/THREESCALE-5827 for the investigation details

Reply:
I looked at the `master` branch in the operator, saw Zync there and assumed it was supported also in 2.9. I'll clarify this :+1:

Reply:
Fixed :+1: